packs/pfs-season-5-bestiary/5-20/asper-hajeri-9-10.json

```diff
@@ -2003,10 +2003,7 @@
                 "base": 24
             },
             "occultism": {
-                "base": 20
-            },
-            "occultism+23": {
-                "base": null
+                "base": 23
```

Comment:
This needs to be checked again the source. 
I've guessed that the occultism bonus is supposed to be 23, rather than the 20 it was, since every other skill modifier is higher than the base version of Asper Hajeri.